modules/nw-metallb-l2padvertisement-cr.adoc

```diff
@@ -0,0 +1,44 @@
+
+:_content-type: CONCEPT
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-configure-address-pools.adoc
+[id="nw-metallb-l2padvertisement-cr_{context}"]
+= About the L2Advertisement custom resource
+
+The fields for the `l2Advertisements` object are defined in the following table:
+
+.L2 advertisements configuration
+[cols="1,1,3a", options="header"]
+|===
+
+|Field
+|Type
+|Description
+
+|`metadata.name`
+|`string`
+|Specifies the name for the L2 advertisement.
+
+|`metadata.namespace`
+|`string`
+|Specifies the namespace for the L2 advertisement.
+Specify the same namespace that the MetalLB Operator uses.
+
+|`spec.ipAddressPools`
+|`string`
+|Optional: The list of IPAddressPools to advertise with this advertisement, selected by name.
+
+|`spec.ipAddressPoolSelectors`
+|`string`
+|Optional: A selector for the IPAddressPools that gets advertised with this advertisement. This is for associating the IPAddressPool to the advertisement based on the label assigned to the IPAddressPool instead of the name itself. If no IPAddressPool is selected by this or by the list, the advertisement is applied to all the IPAddressPools.
+
+|`spec.nodeSelectors`
+|`string`
+|Optional: NodeSelectors allows to limit the nodes to announce as next hops for the LoadBalancer IP. When empty, all the nodes are announced as next hops.
```

Comment:
Node selectors allow you to

load balancer I{

Reply:
The term next hops is new to me, though it seems to be common terminology. Is there a way that better define what that means, however?